open-data/Lesson6-Sharing&Reuse.md

```diff
@@ -0,0 +1,311 @@
+* Biorxiv Preprint server for biology [https://www.biorxiv.org/](https://www.biorxiv.org/)
+* EarthArXiv ([https://eartharxiv.org](https://eartharxiv.org)) and Earth and Space Science Open Archive ([https://essoar.org](https://essoar.org))
+* ASAPbio provides a catalog of preprint servers [https://asapbio.org/preprint-servers](https://asapbio.org/preprint-servers)  
+
+## 6.6 How to evaluate the quality of open data and reuse it
+
+Data reuse is contingent on a number of factors including quality of data, access and reuse conditions, data findability, and more. Below we outline a few essential elements that enable you to assess the relevance and quality of datasets .
+
+### Documentation
+
+Dataset discovery relies on the quality of documentation associated with the datasets, i.e. metadata.  Search engines rely on key information about the datasets which allows them to be indexed and searched,  more on this [https://www.blog.google/products/search/making-it-easier-discover-datasets/](https://www.blog.google/products/search/making-it-easier-discover-datasets/)
+
+No matter how comprehensive the data is, a general rule of thumb is that you should be able to understand what the dataset is about, what is the stage of the data, and what to expect before downloading and processing it.
+
+Some core descriptors to look for are:
+
+* Who created this data?
+* When was it published?
+* How was the data collected?
+* What are the tools used for processing and handling the data?
+* What are the terms for using it?
+* What are the disciplines that used the data?
+* What are the communities that have developed around it?
+
+
+Ideally a fully documented Data Reuse plan should accompany the datasets providing detailed information on What, Who, Where, When and How, see also 5.3 Documenting your Data (Metadata)and Data Reuse Planning Template by mozilla open science
+
+[https://github.com/mozillascience/working-open-workshop/blob/8931bf19bedb99166b931f82dbd2fba815dc6fe1/handouts/data_reuse_plan_template.md](https://github.com/mozillascience/working-open-workshop/blob/8931bf19bedb99166b931f82dbd2fba815dc6fe1/handouts/data_reuse_plan_template.md)
+
+## Formats
+As noted previously, the format of data is key for its interoperability. Data formats are in two classes. Closed, proprietary formats are those where the specification for the way the data is stored is not publicly available and hence is tied explicitly to the software (or hardware) developed by an organization. Open formats have a specification that is available to all (though some may be proprietary and hence subject to conditions that will vary on the license terms applied to them).  Open data formats are cataloged in FAIRsharing.org.
```

Comment:
I do not find the comment on Open Formats very clear, this sounds like it is still proprietary?